web/src/components/workflow-graph/node-base.tsx

```diff
@@ -62,6 +63,39 @@ export default function NodeBase({
 				</Flex>
 			</Card>
 
+			<NodeToolbar
+				isVisible={hasDeletedSecret(nodeId)}
+				position={Position.Top}
+				offset={8}
+				align="end"
+			>
+				<Tooltip content="Missing secret">
+					<Flex
+						style={{
+							transform: "translate(10px, 24px)",
+						}}
+					>
+						<Flex
+							inset="0"
+							className="absolute bg-[var(--red-9)] rounded-full"
+						/>
+						<Flex
+							className="relative rounded-full"
```

Comment:
`position` exists as a prop in `<Flex>`, so please move `absolute` and `relative` out of `className`